frontend/templates/views/partials/format-toggle.j2

```diff
@@ -32,7 +32,12 @@
     {% for format in formats %}
     [% if format != '{{ format }}' %]
     {% do doc.icons.useIcon('icons/amp-' + format + '.svg') %}
-    <a class="ap-m-format-toggle-link ap-m-format-toggle-link-{{ format }}" href="{{ landing_page.url.path }}?format={{ format }}">
+    {% if format in doc.formats %}
+      {% set format_target_url = doc.url.path %}
+    {% else %}
+      {% set format_target_url = landing_page.url.path %}
```

Comment:
We should always link to the same URL (even if the component is not supported). Otherwise the behavior is confusing.

Reply:
The page will show the unsupported format notification which explains the issue to the user.

Reply:
I thought we should not link explicitly to unsupported formats.
But since we use the canonical url there should be no problem with the search engines so I will change this.